nosetimer/plugin.py

```diff
@@ -154,13 +154,19 @@ def report(self, stream):
             with open(self.json_file, 'w') as f:
                 json.dump({'tests': dict((k, v) for k, v in d)}, f)
 
+        totaltime = sum([vv['time'] for kk, vv in d])
```

Comment:
> Please use underscored names: total_time.

Done.



> Also why kk and vv?

The linter complain if I use `k, v` as they are used in the previous list comprehension, and in Python 2 list comprehension don't have their own scope. Thus the 'pep8' groups of tests fails.